pkg/api/types.go

```diff
@@ -133,6 +133,16 @@ const (
 	AgentPoolProfileRoleMaster AgentPoolProfileRole = "master"
 )
 
+// VMSize represents supported VMSizes
```

Comment:
this is probably the "right" thing to do, but one more thing to keep in sync ...

Reply:
As discussed with @kargakis above, we could include azure library for this. But maybe lets start small and lets see.